### DISCUSSION TITLE
Getting Press and Release events

### DISCUSSION THREAD

**Discussion by smallB007** [gyscos/cursive#705] [General]
Hi there,
Is there a way to get separate Press and Release event, like press button and release button?
Thank you

**Comment by gyscos** [comment]
Hi!

As far as I know, the terminal emulator does not send this information to the application for keyboard events (for mouse events, [we do handle mouse click/release events](https://docs.rs/cursive/latest/cursive/event/enum.MouseEvent.html)).

If you know of a way to have the terminal send this information, we may look into supporting it! :)

**Reply by smallB007** [reply]
Hi, thank you for the info. I suspected that as I've scanned through the code and didn't see anything, but thought I better ask ;)
